pkg/register/register.go

```diff
@@ -351,7 +351,7 @@ func getOsReleaseInfo() (map[string]string, error) {
 		val := buf[1]
 
 		if label, ok := envToLabel[env]; ok {
-			data[label] = val
+			data[label] = strings.Trim(val, "\"")
```

Comment:
I found the annotations of the OS versions where including he quotes and were causing scape values to be displayed in the UI.